pkg/config/config.go

```diff
@@ -340,25 +366,51 @@ func (lci *LocalConfigInfo) SetConfiguration(parameter string, value string) (er
 // GetConfiguration uses reflection to get the parameter from the localconfig struct, currently
 // it only searches the componentSettings
 func (lci *LocalConfigInfo) GetConfiguration(parameter string) (interface{}, bool) {
+
+	if strings.ToLower(parameter) == "cpu" {
```

Comment:
why specific handling only for cpu?

Please add it as a comment also please add a note as to what the bool return value stands for
I would be happy if you use a python like structured way of adding function comments for example:

```
// FuncName func does abc
// Parameters:
//    a: a is bla bla
//    b: b is bla bla
// Returns:
//    q: bla bla
//     r: bla bla
```

Reply:
Because setting `cpu` would mean that `mincpu` and `maxcpu` will get set to the same value. It's not a separate parameter but a shortcut.

Reply:
why not memory ?

Reply:
also where is maxcpu set?

Reply:
The issue didn't mention `memory` and here is where the maxcpu is set https://github.com/redhat-developer/odo/blob/69d944f8c6ce5af1c307f847b57f1547254691ea/pkg/config/config.go#L355

Reply:
added the memory parameter